### ISSUE TITLE
Feature Request: Swipe left/right on trackpad to navigate to next/previous article on Mac

### ISSUE THREAD

**Issue by uncleungie** [Ranchero-Software/NetNewsWire#1960]
I'd love to be able to swipe left/right to go to to the next/previous article on the Mac the same way I can in the iOS app. This would be invoked with a two finger swipe on the trackpad when hovering the cursor over the article content.

**Comment by dvincent747** [comment]
Agreed. Ideally it should respect the preference in sysprefs for the number of fingers used for browser back/forward.

**Comment by peternlewis** [comment]
By previous, do you mean the article you were reading before this one (for example, in a different feed if you used Go ➤ Next Unread)?

If so, I came to ask for Go ➤ Previous, so please if this is implemented, ensure there is a menu option for those of us without trackpads. Frequently I'm reading an article and Go ➤ Next Unread and then think I want to re-check something, and there is no way to go back to the previous article. So ideally there would be a Go ➤ Previous, Go ➤ Next (dimmed normally) in a stack of articles, but I'd be happy with even just one or the other.

**Comment by dempseyatgithub** [comment]
I just came here to file this same request. From the user’s perspective, I would expect this to work the same way swiping left and right on an article does on iOS.
